src/query.js

```diff
@@ -744,11 +801,27 @@ function Or() {
  * @return {Expr}
  */
 function Not(boolean) {
+  arity.exact(1, arguments);
   return new Expr({ not: wrap(boolean) });
 }
 
 // Helpers
 
+/**
+ * @ignore
+ */
+function arity(min, max, message, args) {
+  if ((min !== null && args.length < min) || (max !== null && args.length > max)) {
+    throw new errors.InvalidValue(
```

Comment:
On second thought, should we just create another exception class and expose the arity information there, so we don't have to do regex string matching on the message to determine if the correct error was thrown?